package/MDAnalysis/lib/nsgrid.pyx

```diff
@@ -417,7 +417,7 @@ cdef class NSResults(object):
         cdef ns_int idx, nsearch
         cdef real dist2, dist
 
-        nsearch = len(self.searchcoords)
+        nsearch = self.searchcoords.shape[0]
```

Comment:
I am not sure if that should be an issue unless you are looking for a more pythonic/cythonic way of dealing with it. The web search about this yielded [https://github.com/cython/cython/issues/1733](https://github.com/cython/cython/issues/1733). Apparently they have fixed the issue to use len(memoryviews) with a nogil block.

Reply:
Sorry just saw the issue. I guess if we dont want to use the latest cython build, shape would be the way to go